app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitsFormModal/LocalUnitsForm/schema.ts

```diff
@@ -164,21 +165,74 @@ const schema: LocalUnitsFormSchema = {
                                 speciality: {
                                     validations: [lengthSmallerThanCondition(200)],
                                 },
-                                maximum_capacity: {},
-                                number_of_isolation_rooms: {},
+                                maximum_capacity: {
+                                    validations: [
+                                        greaterThanOrEqualToCondition(0),
```

Comment:
We should add an `integerCondition` on fields that are supposed to be integers. It appears that the `maximum_capacity` field can currently accept floating point numbers, which is likely a mistake.

Reply:
Let's add similar validations for all fields.